src/trackers/SHRI.py

```diff
@@ -42,61 +42,66 @@ async def get_name(self, meta):
         name_type = meta.get('type', "")
         source = meta.get('source', "")
         imdb_info = meta.get('imdb_info') or {}
+        type = meta.get('type', "")
+        year = meta['year']
+        title = meta['title']
+
+        akas = imdb_info.get('akas', [])
+        italian_title = None
+
+        remove_list = ['Dubbed']
+        for each in remove_list:
+            shareisland_name = shareisland_name.replace(each, '')
+
+        for aka in akas:
+            if isinstance(aka, dict) and aka.get("country") == "Italy":
+                italian_title = aka.get("title")
+                break
+
+        use_italian_title = self.config['TRACKERS'][self.tracker].get('use_italian_title', False)
+
+        audio_lang_str = ""
+        if not meta.get('language_checked', False):
+            await process_desc_language(meta, desc=None, tracker=self.tracker)
+
+        if meta.get('audio_languages'):
+            audio_languages = []
+            for lang in meta['audio_languages']:
+                lang_up = lang.upper()
+                if lang_up not in audio_languages:
+                    audio_languages.append(lang_up)
+            audio_lang_str = " - ".join(audio_languages)
+
+        audio = meta['audio'].replace('Dual-Audio', '').strip()
+        if meta.get('dual_audio'):
+            shareisland_name = shareisland_name.replace("Dual-Audio", "", 1)
 
         if not meta.get('is_disc') and ('untouched' in basename.lower() or 'vu' in basename.lower()):
-            meta['type'] = 'REMUX'
-            name_type = 'REMUX'
-            
-            # Gestione del titolo italiano
-            title = meta['title']
-            akas = imdb_info.get('akas', [])
-            italian_title = None
-            
-            for aka in akas:
-                if isinstance(aka, dict) and aka.get("country") == "Italy":
-                    italian_title = aka.get("title")
-                    break
-                    
-            use_italian_title = self.config['TRACKERS'][self.tracker].get('use_italian_title', False)
+            type = "REMUX"
+            shareisland_name = shareisland_name.replace(f"{meta['type']}", type, 1)
```

Comment:
Here is how to set some object, and apply it to a name. Also see the type definition at line 45 above.